docs/management/admin/endpoint-command-ref.asciidoc

```diff
@@ -0,0 +1,343 @@
+[[endpoint-command-ref]]
+= {elastic-endpoint} command reference
+
+This page lists the commands for management and troubleshooting of {elastic-endpoint}, the installed component that performs {elastic-defend}'s threat monitoring and prevention.
+
+[NOTE]
+====
+* The service is not added to the `PATH` system variable, so you must prepend the commands with the full OS-dependent path:
+** On Windows: `"C:\Program Files\Elastic\Endpoint\elastic-endpoint.exe"`
+** On macOS: `/Library/Elastic/Endpoint/elastic-endpoint`
+** On Linux: `/opt/Elastic/Endpoint/elastic-endpoint`
+
+* You must run the commands with elevated privileges—as the root user on POSIX systems, or as Administrator on Windows.
+====
+
+The following {elastic-endpoint} commands are available:
+
+* <<elastic-endpoint-diagnostics-command, diagnostics>>
+* <<elastic-endpoint-help-command, help>>
+* <<elastic-endpoint-inspect-command, inspect>>
+* <<elastic-endpoint-install-command, install>>
+* <<elastic-endpoint-memorydump-command, memorydump>>
+* <<elastic-endpoint-run-command, run>>
+* <<elastic-endpoint-send-command, send>>
+* <<elastic-endpoint-status-command, status>>
+* <<elastic-endpoint-test-command, test>>
+* <<elastic-endpoint-top-command, top>>
+* <<elastic-endpoint-uninstall-command, uninstall>>
+* <<elastic-endpoint-version-command, version>>
+
+Each of the commands accepts the following logging options:
+
+* `--log [stdout,stderr,debugview,file]`
+* `--log-level [error,info,debug]`
+
+[discrete]
+[[elastic-endpoint-diagnostics-command]]
+== elastic-endpoint diagnostics
+
+Gather diagnostics information from {elastic-endpoint}. This command produces an archive that contains:
+
+- `version.txt`: Version information
+- `elastic-endpoint.yaml`: Current policy
+- `metrics.json`: Metrics document
+- `policy_response.json`: Last policy response
+- `system_info.txt`: System information
+- `analysis.txt`: Diagnostic analysis report
+- `logs` directory: Copy of {elastic-endpoint} log files
+
+[discrete]
+=== Example
+
+[source,shell]
+------
+sudo /Library/Elastic/Endpoint/elastic-endpoint diagnostics
+------
```

Comment:
The examples only include Linux/macOS. I assume admin-level readers wouldn't struggle with this, but it still feels incomplete to omit Windows.

For the serverless/MDX version, it's pretty easy to create a tabbed widget that includes Windows too ([example](https://www.elastic.co/docs/current/serverless/security/uninstall-agent), [code](https://github.com/elastic/security-docs/blob/main/docs/serverless/edr-install-config/uninstall-agent.mdx?plain=1#L17-L28)), but tabs are kind of a nightmare for AsciiDoc. Maybe we could get away with omitting Windows just for AsciiDoc, or use a simpler soft heading format ([example](https://www.elastic.co/guide/en/security/master/uninstall-agent.html#uninstall-endpoint)).

Reply:
Can't we just drop the prefix? The note at the top of the article was explaining this, so users should figure it out easily what to do when this won't work for them
```
elastic-endpoint diagnostics
```

Reply:
> Can't we just drop the prefix? 

Sounds good! Keeps things simple and clear. Maybe update the note to remind readers they'll need `sudo` to run as the root user? Again, the audience is likely tech-savvy enough to know this, but wouldn't hurt to remind them too.